content/docs/2.0.0/reference-iofogctl/reference-kinds.md

```diff
@@ -13,14 +13,14 @@ metadata:
-| apiVersion         | ioFog YAML schema version. Currently `iofog.org/v2`                                                                                                                                     |
-| kind               | String representing what type of resource we want to deploy. The available values are `ControlPlane`, `Controller`, `Agent`, `AgentConfig`, `Application`, `Microservice` and `Volume`. |
-| metadata           | Object containing metadata about the resource                                                                                                                                           |
-| metadata.name      | User defined, unique identifier of the resource in its namespace.                                                                                                                       |
-| metadata.namespace | Optional. Will force iofogctl to work in this specific namespace (If specified, it overwrites the `-n` CLI option)                                                                      |
-| spec               | Object containing the deployment specifications, different for each resource                                                                                                            |
+| Field              | Description                                                                                                                                                                                                                                                                                                                                                                 |
+| ------------------ | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| apiVersion         | ioFog YAML schema version. Currently `iofog.org/v2`                                                                                                                                                                                                                                                                                                                         |
+| kind               | String representing what type of resource we want to deploy. The available values are `RemoteControlPlane`, `KubernetesControlPlane`, `LocalControlPlane`, `RemoteController`, `KubernetesController`, `LocalController`, `RemoteAgent`, `LocalAgent`, `IofogctlConfig`, `AgentConfig`, `Namespace`, `Registry`, `CatalogItem`, `Application`, `Microservice` and `Volume`. |
```

Comment:
`IofogctlConfig`  and `Namespace` are not a kind that should be exposed to users. It is just the kind we use to store the local namespace information.